float32 axis_0
float32 axis_1
float32 axis_2
bool button_0
bool button_1
bool button_2
bool button_3
